bin/faucet/src/stub_rpc_api.rs

```diff
@@ -0,0 +1,166 @@
+use miden_node_proto::generated::{
+    block::BlockHeader,
+    digest::Digest,
+    requests::{
+        CheckNullifiersByPrefixRequest, CheckNullifiersRequest, GetAccountDetailsRequest,
+        GetAccountProofsRequest, GetAccountStateDeltaRequest, GetBlockByNumberRequest,
+        GetBlockHeaderByNumberRequest, GetNotesByIdRequest, SubmitProvenTransactionRequest,
+        SyncNoteRequest, SyncStateRequest,
+    },
+    responses::{
+        CheckNullifiersByPrefixResponse, CheckNullifiersResponse, GetAccountDetailsResponse,
+        GetAccountProofsResponse, GetAccountStateDeltaResponse, GetBlockByNumberResponse,
+        GetBlockHeaderByNumberResponse, GetNotesByIdResponse, SubmitProvenTransactionResponse,
+        SyncNoteResponse, SyncStateResponse,
+    },
+    rpc::api_server,
+};
+use miden_node_utils::errors::ApiError;
+use tokio::net::TcpListener;
+use tokio_stream::wrappers::TcpListenerStream;
+use tonic::{Request, Response, Status};
+use url::Url;
+
+#[derive(Clone)]
+pub struct StubRpcApi;
+
+#[tonic::async_trait]
+impl api_server::Api for StubRpcApi {
+    async fn check_nullifiers(
+        &self,
+        _request: Request<CheckNullifiersRequest>,
+    ) -> Result<Response<CheckNullifiersResponse>, Status> {
+        unimplemented!();
+    }
+
+    async fn check_nullifiers_by_prefix(
+        &self,
+        _request: Request<CheckNullifiersByPrefixRequest>,
+    ) -> Result<Response<CheckNullifiersByPrefixResponse>, Status> {
+        unimplemented!();
+    }
+
+    #[allow(clippy::unreadable_literal)]
```

Comment:
```suggestion
```
should be able to remove this one by adding `_` to the values. Fairly sure rust-analyzer will do this for you. I'd also recommend using hex values instead `0xABC` (also something rust-analyzer can do).